CLIENT_README.md

```diff
@@ -24,9 +24,9 @@ client = AuthenticatedClient(base_url="https://api.example.com", token="SuperSec
 Now call your endpoint and use your models:
 
 ```python
-from katana_public_api_client.models import MyDataModel
-from katana_public_api_client.api.my_tag import get_my_data_model
-from katana_public_api_client.types import Response
+from katana_public_api_client.generated.models import MyDataModel
```

Comment:
Why are we referencing dummy models? Let's have real examples that use real models from the code base / generated client.